pre-commit: --export-plain-svg is bloating diffs

NikoKrause opened the pull request:
Closes https://github.com/elementary/icons/issues/470

As it is explained in the mentioned issue the option `--export-plain-svg` is reordering elements in the svg file. Doing this twice seems to cancel the effect out.

Reasons to merge this commit:
1. Keeping the git history clean
2. If you added just symbolic links, pre-commit also changed the targeted svg files. With this commit you'll see the symbolic links only, which makes such PRs easier to review.

waldyrious commented on pre-commit:
```diff
@@ -9,6 +9,7 @@ echo "Running Inkscape vacuum. This may take some time..."
 git diff --cached --name-status --diff-filter=ACMR | while read STATUS FILE; do
   if [[ "$FILE" =~ ^.+(svg)$ ]]; then
     inkscape --vacuum-defs -z $CDIR/$FILE --export-plain-svg=$CDIR/$FILE
+    inkscape --vacuum-defs -z $CDIR/$FILE --export-plain-svg=$CDIR/$FILE
```
I'd suggest adding the contents of the commit message as a comment above this line.

NikoKrause replied:
I added a comment.

NikoKrause commented:
I changed the PR. `vacuum-defs` doesn't has to be done twice. Only `export-plain-svg` should be run twice, because that's the command option, which reorders the elements. And I added a comment.

NikoKrause commented:
?

NikoKrause commented:
Can I close this PR or is this waiting to be reviewed?

NikoKrause commented:
Since nobody seem to care, I'll close this.
